### PR TITLE
Create event

### PR DESCRIPTION
**Pull request by liviavinci**

### CODE REVIEW

File: src/scenes/Dashboard/components/TopActions/index.js [hunk 1]
```diff
@@ -14,10 +14,15 @@ import Card from 'material-ui/Card';
 import { withStyles } from 'material-ui/styles';
 
 import dashboardActions from '../../../../redux/Dashboard/actions';
+import CreateEvent from '../../../../scenes/CreateEvent/index';
 import { SortBy } from '../../../../constants';
 import styles from './styles';
 
 class TopActions extends Component {
+  state = {
```
Comment (dwalintukan) [inline src/scenes/Dashboard/components/TopActions/index.js]: does this need to be declared in the constructor?

File: src/scenes/Dashboard/components/TopActions/index.js [hunk 2]
```diff
@@ -33,24 +38,32 @@ class TopActions extends Component {
...
     this.onSortOptionSelected = this.onSortOptionSelected.bind(this);
+    this.onCreateDialogClose = this.onCreateDialogClose.bind(this);
+    this.onCreateDialogOpen = this.onCreateDialogOpen.bind(this);
   }
 
   onSortOptionSelected(event) {
     this.props.sortOrderChanged(event.target.value);
   }
 
+  onCreateDialogClose() {
```
Comment (dwalintukan) [inline src/scenes/Dashboard/components/TopActions/index.js]: use fat arrow function. Alex showed me this recently and seems handy so we dont have to remember to bind it.
`onCreateDialogClose = () => {
}`

File: src/scenes/Dashboard/components/TopActions/index.js [hunk 3]
```diff
@@ -33,24 +38,32 @@ class TopActions extends Component {
...
+    this.setState({ createDialogOpen: false });
+  }
+
+  onCreateDialogOpen() {
```
Comment (dwalintukan) [inline src/scenes/Dashboard/components/TopActions/index.js]: use fat arrow function

File: src/scenes/CreateEvent/index.js [hunk 4]
```diff
@@ -0,0 +1,236 @@
...
+                  fullWidth
+                  type="input"
+                  placeholder="Block Number"
+                />
+              </Grid>
+            </Grid>
+          </Grid>
+          <Grid container>
+            <Grid item xs={3}>
+              RESULT END TIME
```
Comment (dwalintukan) [inline src/scenes/CreateEvent/index.js]: missing Result Set Start Time

File: src/scenes/CreateEvent/index.js [hunk 5]
```diff
@@ -0,0 +1,236 @@
...
+                fullWidth
+                multiline
+                rows="3"
+                placeholder="Introduction (opitonal)"
+              />
+            </Grid>
+          </Grid>
+          <Grid container>
+            <Grid item xs={3}>
+              PREDICTION END TIME
```
Comment (dwalintukan) [inline src/scenes/CreateEvent/index.js]: missing Prediction Start Time

File: src/scenes/CreateEvent/index.js [hunk 6]
```diff
@@ -0,0 +1,236 @@
...
+                + Add Result
+              </Button>
+            </Grid>
+          </Grid>
+          <Grid container>
+            <Grid item xs={3}>
+              RESULT SETTER
+            </Grid>
+            <Grid item xs={9}>
+              <Input
```
Comment (dwalintukan) [inline src/scenes/CreateEvent/index.js]: we also need an address selector for the Result Setter

File: src/scenes/CreateEvent/index.js [hunk 7]
```diff
@@ -0,0 +1,236 @@
...
+import moment from 'moment';
+import Web3Utils from 'web3-utils';
+
+import SelectAddressDialog from '../../components/SelectAddressDialog/index';
+import graphqlActions from '../../redux/Graphql/actions';
+import appActions from '../../redux/App/actions';
+import { calculateBlock } from '../../helpers/utility';
+import { defaults } from '../../config/app';
+
+class CreateEvent extends React.Component {
```
Comment (dwalintukan) [inline src/scenes/CreateEvent/index.js]: CreateEventDialog?

**Comment by liviavinci** [comment]
oh btw this is not ready for review lol

**Pull request closed without merging**